os/mm/mm_heap/mm_manage_allocfail.c

```diff
@@ -99,6 +105,30 @@ void mm_manage_alloc_fail(struct mm_heap_s *heap, int startidx, int endidx, size
 	abort_mode = true;
 #endif
 
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
+	struct watchdog_status_s status;
```

Comment:
We can rename all our variables like status -> wd_status, so that it indicates relation to wdog.

Reply:
Thank you for review. applied